### DISCUSSION TITLE
Optimization trouble.

### DISCUSSION THREAD

**Discussion by nyakokitsu** [ppy/osu#26820] [Q&A]
### Type

Performance

### Bug description

Sup, when I play Laser, it freezes for a split second, after which the game returns to normal. This is very unsettling during the game. I play osu!std, linux ubuntu 22, x11

### Version

2024.130.2

### Logs

none

**Comment by bdach** [comment]
Please attach video of this with frame graph shown (ctrl-f11 twice).

**Reply by nyakokitsu** [reply]
OBS crashes when i starting osu. 🥹

**Reply by Stoppedpuma** [reply]
You can try using ffmpeg as its more lightweight https://trac.ffmpeg.org/wiki/Capture/Desktop
